### PR TITLE
PRC-323: Add address select type

### PR DESCRIPTION
**Pull request by james-baker-moj**
Initial part of the add address journey to select address type.

### CODE REVIEW

File: server/routes/contacts/manage/addresses/address-type/addressTypeController.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    )
+  }
+
+  private getSelectedOptions(
+    options: ReferenceCode[],
+    selected?: string,
+  ): Array<{
+    value: string
+    text: string
+    selected?: boolean
```
Comment (timharrison-moj) [inline server/routes/contacts/manage/addresses/address-type/addressTypeController.ts:65]: Although in NOMIS addressType is nullable, do we definitely want to force a selection in our journeys?

File: server/views/pages/contacts/manage/address/addressType.njk [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <div class="govuk-grid-column-three-quarters">
+            <form method='POST'>
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+                {% set defaultOptions = [
+                    { divider: 'or' },
+                    {
+                        value: "DO_NOT_KNOW",
+                        text: "I don't know",
+                        checked: journey.addressType === 'DO_NOT_KNOW'
+                    }
```
Comment (timharrison-moj) [inline server/views/pages/contacts/manage/address/addressType.njk:27]: Ah - there is an I do not know option..

**Review by timharrison-moj — APPROVED** [review]
LGTM

File: server/views/pages/contacts/manage/address/addressType.njk (hunk 2 above)
Comment (james-baker-moj) [reply server/views/pages/contacts/manage/address/addressType.njk:27]: Yeah, that's just to make them choose a real option or DO_NOT_KNOW. It'll translate to null when we call the API

**Pull request merged**